contracts/Registry.vy

```diff
@@ -1000,3 +1165,35 @@ def set_liquidity_gauges(_pool: address, _liquidity_gauges: address[10]):
             self.liquidity_gauges[_pool][i] = ZERO_ADDRESS
         else:
             break
+    self.last_updated = block.timestamp
+
+
+@external
+def set_pool_asset_type(_pool: address, _asset_type: String[32]):
```

Comment:
Love this.  Having the flexibility to modify after the fact without removing and re-adding the whole pool is wonderful.

How do you feel about not classifying with `String`, though?  We can instead use integers as a sort-of type ID that corresponds to a list of types off-chain.  Would mean one less write to storage and I don't think anybody is going to need to see the exact type on-chain.